content/overview/get-started/gs-ingress.md

```diff
@@ -4,24 +4,32 @@ uid: gs-ingress
 
 # Step 4: Ingress data
 
-Ingress data into your AVEVA Data Hub tenant using one of two options, or both:
+Ingress data into your AVEVA Data Hub tenant using any of the following options.
 
 ## Option A: Collect data from PI Server
 
-**Prerequisite:** You must have a PI Server installed and configured.
+Transfer on-premises PI data and Asset Framework (AF) data into AVEVA Data Hub via a PI to Data Hub Agent.
 
-1. <xref:install-agent>
-2. <xref:pi-to-ocs-utility>
-3. <xref:create-transfer>
+1. <xref:install-agent>.
+
+1. <xref:pi-to-ocs-utility>.
+
+1. <xref:create-transfer>.
 
 ## Option B: Collect data directly from edge devices
 
-Review the following documentation and complete the following tasks:
+Collect data from edge devices using an AVEVA Data Hub client and OMF connection.
 
 **Prerequisite:** You must have EDS or an adapter installed and configured on an edge device.
 
-1. [Review OMF best practices](xref:bpOMFConnection)
-2. [Create a client credential client](https://docs.aveva.com/bundle/data-hub/page/set-up/clients/client-credentials-client-procedure.html)
- The edge device uses this client to authenticate with AVEVA Data Hub.
-3. [Create an OMF connection](https://docs.aveva.com/bundle/data-hub/page/add-organize-data/collect-data/connectors/omf/omf-connection-procedure.html)
- Connect your edge devices to AVEVA Data Hub.
+1. Review <xref:bpOMFConnection>.
+
+1. <xref:gpClientCredentialsClient>. The edge device uses this client to authenticate with AVEVA Data Hub.
+
+1. <xref:gpOMFConnection>. Connect your edge devices to AVEVA Data Hub.
+
+## Option C: Ingress data programmatically
+
+APIs provide programmatic access to bring sequential data into AVEVA Data Hub. This can be done through custom OMF applications or utilizing SDS data writes.
+
+See <xref:omfVsSds>.
```

Comment:
It seems like we should link to content/developer-guide/sequential-data-store-dev/sds-write-data.md here and then make the omfVsSds topic a note.